Jenkinsfile

```diff
@@ -0,0 +1,67 @@
+// List of targets to compile
+def morpheusTargets = [
```

Comment:
Shouldn't we use just regular ``targets`` ? prefix ``morpheus`` shall be removed?
Why there are some targets commented out ? Should not be there ?